src/Ombi/ClientApp/src/app/media-details/media-details.component.scss

```
@@ -1,376 +1,751 @@
-        margin-left:0px;
-    }
+.left-panel-details {
+    font-weight: 100;
+}
 
-    .media-row .btn-spacing{
-        margin-right:0;
-    }
+.genre-button-container .mat-chip-list .mat-chip-list-wrapper {
+    margin-top: 3px;
+    margin: 0;
```

Stylelint (17.6.0)

[error] Unexpected shorthand "margin" after "margin-top" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)